node/src/client.rs

```diff
@@ -21,11 +21,25 @@ use std::sync::Arc;
 pub type FullClient<RuntimeApi, Executor> =
 	TFullClient<Block, RuntimeApi, NativeElseWasmExecutor<Executor>>;
 pub type FullBackend = TFullBackend<Block>;
-type PicassoClient = FullClient<picasso_runtime::RuntimeApi, PicassoExecutor>;
+pub(crate) type PicassoClient = FullClient<picasso_runtime::RuntimeApi, PicassoExecutor>;
 #[cfg(feature = "composable")]
-type ComposableClient = FullClient<composable_runtime::RuntimeApi, ComposableExecutor>;
+pub(crate) type ComposableClient = FullClient<composable_runtime::RuntimeApi, ComposableExecutor>;
 #[cfg(feature = "dali")]
-type DaliClient = FullClient<dali_runtime::RuntimeApi, DaliExecutor>;
+pub(crate) type DaliClient = FullClient<dali_runtime::RuntimeApi, DaliExecutor>;
+
+/// A client instance of Picasso.
+#[derive(Clone)]
+pub enum Client {
+	/// Picasso client type
+	Picasso(Arc<PicassoClient>),
```

Comment:
So Picasso becomes our base?

Reply:
Perhaps it is more logical to have composable be that base, as it should have the least amount of features. Then `picasso` is additive, and `dali` is basically `full`.